chain/network/src/peer_manager/peer_store.rs

```diff
@@ -792,6 +786,60 @@ mod test {
                 Blacklist::from_iter([format!("{}", peer_infos[2].addr.unwrap())].into_iter());
             let _peer_store = PeerStore::new(store.clone(), &[], blacklist).unwrap();
         }
-        assert_peers(tmp_dir.path(), &peer_ids[0..2]);
+        assert_peers_in_store(tmp_dir.path(), &peer_ids[0..2]);
+    }
+
+    fn assert_peers_in_store(store_path: &std::path::Path, expected: &[PeerId]) {
+        let store = create_store(store_path);
+        let stored_peers: HashSet<PeerId> = HashSet::from_iter(
+            store.iter(DBCol::Peers).map(|(key, _)| PeerId::try_from_slice(key.as_ref()).unwrap()),
+        );
+        let expected: HashSet<PeerId> = HashSet::from_iter(expected.iter().cloned());
+        assert_eq!(stored_peers, expected);
+    }
+
+    fn assert_peers_in_cache(
+        peer_store: &PeerStore,
+        expected_peers: &[PeerId],
+        expected_addresses: &[SocketAddr],
+    ) {
+        let expected_peers: HashSet<&PeerId> = HashSet::from_iter(expected_peers.iter());
```

Comment:
Not important to change but here and a few lines below the `iter` call is unnecessary:
```suggestion
        let expected_peers: HashSet<&PeerId> = HashSet::from_iter(expected_peers);
```

Reply:
Removed - I hope I haven't missed any unnecessary calls.